bidsify.py

```diff
@@ -1,20 +1,76 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
 """
 From https://github.com/BIDS-Apps/example/blob/aa0d4808974d79c9fbe54d56d3b47bb2cf4e0a0d/run.py
 """
 import os
-import os.path as op
 import tarfile
+import pathlib
 import argparse
 import subprocess
-
+import shutil
 import pydicom
 import numpy as np
 import pandas as pd
 from dateutil.parser import parse
 
+def manage_dicom_dir(dicom_dir):
```

Comment:
Please use two empty lines before function definitions.

Reply:
Fixed the line spacing

Reply:
I'm not seeing it. It looks like it still has only one empty line here.

Reply:
Weird, check it again